docs/extensibility/datasources-section.md

```diff
@@ -0,0 +1,85 @@
+# _dataSources_ section
+
+The optional **dataSources** section contains an object that maps a data source name to a data source configuration object. The data source name preceded by a dollar sign '\$' is used in the **source** expression.
+
+Data sources are provided in all [JSONata](https://docs.jsonata.org/overview.html) formulas as functions to call. For example, `{ "source": $myRelatedResource().metadata.labels }` returns the `metadata.labels` of the related resource.
+
+Since the whole request is being provided, individual resources can be accessed using the `items` field, for example `{ "widget": "Table", "source": "$myRelatedResources().items" }`.
```

Comment:
```suggestion
Because the whole request is being provided, individual resources can be accessed using the `items` field, for example `{ "widget": "Table", "source": "$myRelatedResources().items" }`.
```
- "since" is [ambiguous](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/5b157482b3fd4e389e3aa283f86480ef.html?state=DRAFT&locale=en-US)
- can we switch the passive phrases to more active voice?